frontend/appflowy_tauri/src/appflowy_app/utils/change_notifier.ts

```diff
@@ -1,17 +1,29 @@
-import { Subject } from 'rxjs';
+import { Observable, Subject } from 'rxjs';
 
 export class ChangeNotifier<T> {
+  private isUnsubscribe = false;
   private subject = new Subject<T>();
 
   notify(value: T) {
     this.subject.next(value);
   }
 
-  get observer() {
+  get observer(): Observable<T> | null {
+    if (this.isUnsubscribe) {
+      return null;
+    }
     return this.subject.asObservable();
   }
 
+  // Unsubscribe the subject might cause [UnsubscribedError] error if there is
```

Comment:
Maybe we don't need to add judgment here, just need to ensure the execution of the `subscription.unsubscribe();`